### ISSUE TITLE
eeg_matchingpennies, remove git-annex version?

### ISSUE THREAD

**Issue by sappelhoff** [mne-tools/mne-bids-pipeline#507]
I am in the process of reorganizing the eeg_matchingpennies dataset (once again ...) in https://github.com/bids-standard/bids-examples/pull/265 - there is some information about why in https://github.com/mne-tools/mne-bids/issues/936#issuecomment-1006833527

When I am done there, I'd like to push these changes to the "full data storage" on OSF: https://osf.io/cj2dr/

Usually that means that I have to update the git-annex mirror at https://github.com/sappelhoff/eeg_matchingpennies which is always a bit hacky due to the way I implemented this (back then, datalad didn't have OSF integration, so this is based on a lot of manual steps and running a bunch of scripts in the right order).

Long story short: Would it be a problem if I just removed the git-annex mirror? --> the `eeg_matchingpennies` dataset could still be part of the pipeline via OSF, like "ERPCore":

https://github.com/mne-tools/mne-bids-pipeline/blob/ca17542efd233566fa80d7ecc1ec392add36073e/tests/datasets.py#L20-L25

cc @hoechenberger

**Comment by agramfort** [comment]
No objection on my side

Keep it simple

**Comment by hoechenberger** [comment]
Like you pointed out, we already have OSF support in our testing infrastructure, so I don't see any need to keep the git mirror.
